clib/feldspar_c99.c

```diff
@@ -26,9 +26,11 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 
+#define __STDC_FORMAT_MACROS
```

Comment:
This should only apply to C++ as far as I can tell.

Reply:
I think you're right. That was a bit of overkill.